app/actions/views/more_dms.js

```diff
@@ -21,7 +21,7 @@ export function makeDirectChannel(otherUserId) {
         if (channel && myMembers[channel.id]) {
             result = {data: channel};
 
-            toggleDMChannel(otherUserId, 'true')(dispatch, getState);
+            toggleDMChannel(otherUserId, 'true', channel.id)(dispatch, getState);
```

Comment:
can you please change this to `dispatch(toggleDMChannel(otherUserId, 'true', channel.id))`

Reply:
Will do